### DISCUSSION TITLE
Great job!

### DISCUSSION THREAD

**Discussion by cukal** [tesla-android/issue-tracker#18] [Q&A]
Up & running! 
Congrats on the amazing job, must have been quite the ride dreaming up the whole approach and executing it while taking so many hurdles!

I'm using wireless CarPlay with CarlinKit CPC200-Autokit and it seems there is no way to persuade the setup into pushing audio from Waze to the car. When in CarPlay mode Waze output is greyed out & defaults to _"Device Default - CarPlay is connected"_ so it seems impossible to connect Waze audio to Bluetooth when the phone is in CarPlay mode (or I haven't found it) and the car just does not connect to CPC200-Autokit in box mode. When trying to pair the car with Autolink Box it freezes for a while. Blue led does blink, but never pairs, eventually becomes responsive again. Spotify output seems to bypass CarPlay (you can select it in the iOS player output) and that works. 
Is this correct?

Thanks!

**Comment by mikegapinski** [comment]
The fix for navigation audio is being tested currently, it's more of a proof of concept now but it'll work with Bluetooth as a audio channel selected in AutoKit. This is an issue with AutoKit to begin with. Even when BT is selected as an audio source the navigation audio is pushed via Android...
